src/ngraph/util.cpp

```diff
@@ -14,6 +14,7 @@
 * limitations under the License.
 *******************************************************************************/
 
+#include <algorithm>
```

Comment:
Need to also include `<numeric>` for `iota`. This is one of those files that some compilers implicitly include and some don't.